### DISCUSSION TITLE
Issue with go generate, complaining about file missing in go.sum

### DISCUSSION THREAD

**Discussion by sampath-od** [99designs/gqlgen#3180] [General]
Help me find a solution for this. Using go 1.22

../../pkg/mod/github.com/99designs/gqlgen@v0.17.49/main.go:17:2: missing go.sum entry for module providing package github.com/urfave/cli/v2 (imported by github.com/99designs/gqlgen); to add:
        go get github.com/99designs/gqlgen@v0.17.49
graph/resolver.go:3: running "go": exit status 1

**need a fix for this**. 
 go get github.com/99designs/gqlgen@v0.17.49 is adding the file.
 go mod tidy is removing the file.

**Comment by sampath-od** [comment]
Can anyone please assist ?

**Reply by JonasDoe** [reply]
Does it work when you install `gqlgen` with `go install github.com/99designs/gqlgen@latest`? You can run it by `gqlgen` if the `GOBIN` (default: `~/go/bin`) is in your `PATH` env variable (or by invoking it directly with `~/go/bin/gqlgen`), while you're in the project directory.
